include/seqan3/argument_parser/validators.hpp

```diff
@@ -612,25 +624,41 @@ class output_file_validator : public file_validator_base
      * \{
      */
 
-    //!\copydoc seqan3::input_file_validator::input_file_validator()
-    output_file_validator()
+    output_file_validator() = delete; //!< Deleted.
+    output_file_validator(output_file_validator const &) = default; //!< Defaulted.
+    output_file_validator(output_file_validator &&) = default; //!< Defaulted.
+    output_file_validator & operator=(output_file_validator const &) = default; //!< Defaulted.
+    output_file_validator & operator=(output_file_validator &&) = default; //!< Defaulted.
+    virtual ~output_file_validator() = default; //!< Virtual Destructor.
+
+    /*!\brief Constructs from a given collection of valid extensions.
+     * \param[in] mode The property whether it is valid to overwrite the output file.
+     *
+     * \details
+     *
+     * This constructor is only available if `file_t` does not name a valid seqan3 file type that contains a
+     * static member `valid_formats`.
+     */
+    explicit output_file_validator(output_file_open_options const mode) : file_validator_base{}, mode{mode}
     {
         if constexpr (!std::same_as<file_t, void>)
             file_validator_base::extensions = detail::valid_file_extensions<typename file_t::valid_formats>();
     }
 
-    output_file_validator(output_file_validator const &) = default;               //!< Defaulted.
-    output_file_validator(output_file_validator &&) = default;                    //!< Defaulted.
-    output_file_validator & operator=(output_file_validator const &) = default;   //!< Defaulted.
-    output_file_validator & operator=(output_file_validator &&) = default;        //!< Defaulted.
-    virtual ~output_file_validator() = default;                                   //!< Virtual Destructor.
-
-    //!\copydoc seqan3::input_file_validator::input_file_validator(std::vector<std::string>)
-    explicit output_file_validator(std::vector<std::string> extensions)
+    /*!\brief Constructs from a given collection of valid extensions.
+     * \param[in] extensions The valid extensions to validate for.
+     * \param[in] mode The property whether it is valid to overwrite the output file.
+     *
+     * \details
+     *
+     * This constructor is only available if `file_t` does not name a valid seqan3 file type that contains a
+     * static member `valid_formats`.
+     */
+    explicit output_file_validator(output_file_open_options const mode, std::vector<std::string> extensions)
```

Comment:
```suggestion
    explicit output_file_validator(output_file_open_options const mode, std::vector<std::string> extensions = {})
```

Does this work, if yes, can you merge this constructor with the above one?

Reply:
No, it looks like this isn't working:
```bash
.../seqan3/test/unit/argument_parser/format_parse_validators_test.cpp:264:112:
    error: no matching function for call to 'seqan3::output_file_validator<dummy_file>::output_file_validator(<brace-enclosed initializer list>)'
264 |             seqan3::output_file_validator<dummy_file> my_validator{seqan3::output_file_open_options::create_new};
```